### ISSUE TITLE
Can't connect to telegram api.

### ISSUE THREAD

**Issue by LinArcX** [wangyi2005/ng-hr#2]
Hi. i'm using [pyrogram](https://github.com/pyrogram/pyrogram) to create telegram bots. unfortunatley telegram is censord in my country. so i decide to use v2ray and deploy it on heroku using your awesome repo. until now everything is ok. i can go through blocked site easily in my browser.
But if i want to develop telegram bot using pyrogram, [i should create a proxy file](https://docs.pyrogram.ml/resources/SOCKS5Proxy).
It is my config.ini file:
```
[proxy]
enabled = True
hostname = 127.0.0.1
port = 1080
```
But it can't connect to telegram. i don't know why. i ask same question in pyrogram chatroom and the said me that i should another address instead of localhost and setup a socks5 proxy on my machine.
It's my config.json and as you can see, i enable socks for it:
https://gist.github.com/0b73024a0889850fb318a870d64036b2

**Comment by wangyi2005** [comment]
v2ray log？

**Comment by kerlingcode** [comment]
ws可以配置host伪装吗？想问一下

**Comment by LinArcX** [comment]
@wangyi2005 It doesn't work anymore.
This is the output of access log:
https://gist.github.com/LinArcX/e463223a9dd9e21d0484c2f6ced3b422

**Comment by wangyi2005** [comment]
check v2ray  and web browser config
socks5 proxy 127.0.0.1:1080
http proxy 127.0.0.1:8080

**Comment by LinArcX** [comment]
@wangyi2005 Yeah, it's my fault sorry. i change socks to http in firefox extension and now it works in browser. 
Edit: Logs updated.

**Comment by LinArcX** [comment]
Some one in pyrogram chatroom want me to ping these ips. but i can't ping them at all:
1: "149.154.175.50",
2: "149.154.167.51",
3: "149.154.175.100",
4: "149.154.167.91",
5: "91.108.56.149"
Is this related to my issue?

**Comment by wangyi2005** [comment]
sorry , i have no idea about how  pyrogram  to use socks5 proxy.

**Comment by LinArcX** [comment]
what about logs?
